server/handlebars/embed/public/style.css

```diff
@@ -41,45 +41,77 @@ h3#report-title {
     font-size: 14px;
     margin-bottom: 1em;
     text-align: center;
-}
-
-#embed-report-phase-container {
+  }
+  
+  #embed-report-phase-container {
     margin-bottom: 1em;
-}
-#embed-report-phase-container summary:focus-visible {
+  }
+  #embed-report-phase-container summary:focus-visible {
     outline-offset: -2px;
     outline: 2px solid #3a86d1;
-}
-
-#candidate-title {
-    border: 1.5px solid #ffc4a2;
+  }
+  
+  #embed-report-phase-container summary::after {
+    width: 30px;
+    height: 30px;
+    border-radius: 3px;
+    display: block;
+    content: '';
+    background-position: center center;
+    background-repeat: no-repeat;
+    position: absolute;
+    right: 0.5em;
+    top: 0.5em;
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='12' height='6.861' viewBox='0 0 12 6.861'%3E%3Cpath id='Icon_ionic-ios-arrow-down' data-name='Icon ionic-ios-arrow-down' d='M12.19,16.039,16.727,11.5a.854.854,0,0,1,1.211,0,.865.865,0,0,1,0,1.215L12.8,17.858a.856.856,0,0,1-1.183.025L6.438,12.717A.858.858,0,1,1,7.649,11.5Z' transform='translate(18.188 18.108) rotate(180)' fill='%2360470c'/%3E%3C/svg%3E%0A");
+    background-color: #fce1a4;
+    transform: rotate(180deg);
+  }
+  
+  #embed-report-phase-container[open] summary::after {
+    transform: rotate(0deg);
+  }
+  
+  /* section: CANDIDATE WARNING STYLING */
+  summary {
+    display: block;
+  }
+  
+  #candidate-title {
+    background-color: #fdefce;
+    border: 1.5px solid #fce1a4;
     border-top-left-radius: 3px;
     border-top-right-radius: 3px;
-    background-color: #ffd8c1;
     padding: 0.5em 0.5em 0.5em 1em;
     cursor: pointer;
-}
-
-#candidate-title > span {
+    --space-between-elements: 1em;
+    --left-right-padding: 55px;
+  }
+  
+  #candidate-title > * {
```

Comment:
I think instead of applying this to all the children, make this individual margin left and right on the span and icon. I also think that the space between the icon and span can be smaller.

Reply:
This has been taken care of.